ci/Dockerfile

```diff
@@ -81,8 +81,8 @@ RUN wget https://github.com/Kitware/CMake/releases/download/v3.22.1/cmake-3.22.1
 ENV PATH="/opt/cmake/bin:${PATH}"
 
 # Installing Golang
-RUN GOLANG_SHA256="698ef3243972a51ddb4028e4a1ac63dc6d60821bf18e59a807e051fee0a385bd" \
- && GOLANG_TARBALL="go1.20.4.linux-amd64.tar.gz" \
+RUN GOLANG_SHA256="d0398903a16ba2232b389fb31032ddf57cac34efda306a0eebac34f0965a0742" \
+ && GOLANG_TARBALL="go1.21.0.linux-amd64.tar.gz" \
```

Comment:
Please use `1.21.10` to match what we used for CI hosts:

- [`infra-role-golang#5c8083e8`](https://github.com/status-im/infra-role-golang/commit/5c8083e8) - upgrade from 1.20.12 to 1.21.10

And remember to update the image tag for Linux builds.

Reply:
Done! Thanks for the feedback.